src/api-v3/paths/accounts/{address}/settings.spec.ts

```diff
@@ -39,19 +41,52 @@ describe.skip(path, () => {
       .end(done);
   });
 
-  // it('GET - fails validation when response is invalid', (done) => {
-  //   sinon.stub(rippleApi, 'isConnected').returns(true);
-  //   const getAccountInfoResponse = Object.assign({}, getAccountInfoFixture);
-  //   // For testing only:
-  //   (getAccountInfoResponse.account_data as any).foo = 'bar'; // eslint-disable-line @typescript-eslint/no-explicit-any
-  //   sinon.stub(rippleApi, 'request').resolves(getAccountInfoResponse);
-
-  //   request(mockApp)
-  //     .get(path)
-  //     .expect(200)
-  //     .expect(() => {
-  //       expect(capture(mockedDebuglog.log).first()).to.deep.equal(["\u001b[31m%s\u001b[0m","/accounts/{address}/info: validation:",{"message":"The response was not valid.","errors":[{"path":"account_data","errorCode":"additionalProperties.openapi.responseValidation","message":"account_data should NOT have additional properties"}]}]);
-  //     })
-  //     .end(done);
-  // });
+  it.skip('GET settings - request fails validation when request is invalid', (done) => {
+    sinon.stub(rippleApi, 'isConnected').returns(true);
+    const invalidGetSettingsResponse = {
+      foo: 'bar'
+    }
+    sinon.stub(rippleApi, 'request').resolves(invalidGetSettingsResponse);
+
+    request(mockApp)
+      .get(path)
+      // .expect(200)
```

Comment:
Consider erasing some of this? 
- No need to add an `expect` that is commented out
- console.log could be removed if the test isn't being run